presto-mongodb/src/test/java/io/prestosql/plugin/mongodb/TestMongoDistributedQueriesLatest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.mongodb;
+
+import com.google.common.collect.ImmutableMap;
+import io.prestosql.testing.QueryRunner;
+import io.prestosql.tpch.TpchTable;
+
+import static io.prestosql.plugin.mongodb.MongoQueryRunner.createMongoQueryRunner;
+
+public class TestMongoDistributedQueriesLatest
```

Comment:
Assuming that we have two latests classes, let's call it `TestMongo3DistributedQueriesLatest`.